<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>711</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="_1WorldConfiguration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>441</height>
      </rect>
     </property>
     <attribute name="label">
      <string>World Configuration</string>
     </attribute>
     <widget class="QSpinBox" name="sBox_worldc_sizeX">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>30</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="sBox_worldc_sizeY">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_applyWorldConfigSize">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Set world size</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="_2RobotConfiguration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>441</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Robot Configuration</string>
     </attribute>
     <widget class="QPushButton" name="btnCreateRobot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Robot</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_robot_xPos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_robot_yPos">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_robot_radius">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="input_robot_detRadius">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position on X-axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Position on Y-axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>80</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Radius / Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Detection radius</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>121</width>
        <height>120</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSpinBox" name="input_robot_color_b">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSpinBox" name="input_robot_color_r">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="input_robot_color_g">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSpinBox" name="input_robot_rotation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-360</number>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>95</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Default rotation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTest_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="_3ObstacleConfigaration">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>441</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Obstacle Configaration</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1243</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
